src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/AppointmentsService.kt

```diff
@@ -54,26 +55,44 @@ class AppointmentsService(
     return actionPlanAppointmentRepository.save(appointment)
   }
 
-  fun getAppointments(actionPlanId: UUID): List<ActionPlanAppointment> {
+  fun updateAppointmentWithAttendance(
```

Comment:
Should the business tier not also reflect the business action being done, i.e. recordAttendance

Reply:
It probably should. Will change